indico/modules/events/papers/views.py

```diff
@@ -59,8 +60,16 @@ def render_paper_page(paper, view_class=None):
     review_form = None
     reviewed_for_groups = list(paper.last_revision.get_reviewed_for_groups(session.user))
     if len(reviewed_for_groups) == 1:
-        review_form = build_review_form(paper, reviewed_for_groups[0])
-    params = {'paper': paper, 'comment_form': comment_form, 'review_form': review_form}
+        review_form = build_review_form(paper.last_revision, reviewed_for_groups[0])
+    judgment_form = PaperJudgmentForm()
```

Comment:
pass `formdata=None` to all the forms here to ensure nothing is prefilled if this function is used on a page that has POST data for a field matching the name of one of the fields of the form